src/client/renderer/r_lightmap.c

```diff
@@ -62,74 +56,23 @@ static r_image_t *R_AllocLightmap_(r_image_type_t type) {
 	image->media.Free = R_FreeLightmap;
 
 	image->type = type;
-	image->width = image->height = r_lightmap_state.block_size;
+	image->width = width;
+	image->height = height;
 
 	return image;
 }
 
-#define R_AllocLightmap() R_AllocLightmap_(IT_LIGHTMAP)
-#define R_AllocDeluxemap() R_AllocLightmap_(IT_DELUXEMAP)
-
-/**
- * @brief
- */
-static void R_UploadLightmapBlock(r_bsp_model_t *bsp) {
-
-	R_UploadImage(r_lightmap_state.lightmap, GL_RGB, r_lightmap_state.sample_buffer);
-	R_UploadImage(r_lightmap_state.deluxemap, GL_RGB, r_lightmap_state.direction_buffer);
```

Comment:
Do we lose the ability to upload lightmaps as GL_RGB and not GL_RGBA with this change?